src/main/java/frc/robot/GameRobotContainer.java

```diff
@@ -75,8 +64,7 @@ public GameRobotContainer() {
         m_ArmSubsystem = new ArmSubsystem(ArmSubsystem.Arm.ARM_ID, "rio", 
                         ArmSubsystem.Arm.ENCODER_DIO_SLOT);
 
-        m_PizzaBoxSubsystem = new PizzaBoxSubsystem(PizzaBoxSubsystem.PIZZABOX_ID, 
-                    "rio", PizzaBoxSubsystem.SERVO_PWN_SLOT, PizzaBoxSubsystem.SERVO2_PWN_SLOT);
+        m_PizzaBoxSubsystem = new PizzaBoxSubsystem(PizzaBoxConstants.PIZZA_BOX_ID, "rio", PizzaBoxConstants.SERVO_PWD, PizzaBoxConstants.FLAP_PWD);
```

Comment:
I think this was bad practice from last season. Since the IDs and CAN doesn't change, we do not need to these parameters in the constructor. You can access those constants in the subsystem directly.

Makes the code cleaner.